portfolio/src/main/webapp/index.html

```diff
@@ -3,16 +3,106 @@
   <head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
+    <link rel="stylesheet" href="node_modules/bulma/css/bulma.min.css">
     <link rel="stylesheet" href="style.css">
+    <script defer src="https://use.fontawesome.com/releases/v5.3.1/js/all.js"></script>
+    <script src="node_modules/jquery/dist/jquery.min.js"></script>
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <section class="hero is-light is-large">
+        <div class="hero-head">
+    <nav class="navbar">
+      <div class="container">
+        <div class="navbar-brand">
+          <a class="navbar-item">
+            <h1>Iris Liu</h1>
+          </a>
+          <span class="navbar-burger burger" data-target="navbarMenuHeroB">
+            <span></span>
+            <span></span>
+            <span></span>
+          </span>
+        </div>
+        <div id="navbarMenuHeroB" class="navbar-menu">
+          <div class="navbar-end">
+            <a class="navbar-item is-active" href="index.html">
+              Home
+            </a>
+            <a class="navbar-item" href="me.html">
+              About Me
+            </a>
+            <a class="navbar-item" href="photo.html">
+              Photography
+            </a>
+            <span class="navbar-item">
+              <a class="button is-inverted navbar-icon-background" href="https://linkedin.com/in/jiayi-liu-1010" target="_blank">
+                <span class="icon has-text-dark is-large">
+                  <i class="fab fa-linkedin"></i>
+                </span>
+              </a>
+            </span>
+            <span class="navbar-item">
+              <a class="button is-inverted navbar-icon-background" href="https://github.com/irisliu77" target="_blank">
+                <span class="icon has-text-dark is-large">
+                  <i class="fab fa-github-square"></i>
+                </span>
+              </a>
+            </span>
+          </div>
+        </div>
+      </div>
+    </nav>
+  </div>
+
+  <div class="hero-body"> 
+    <div class="columns is-centered">
+       <div class="column is-one-third">
+           <div class="container has-text-centered">
+              <img id="homepage_photo" src="images/homepage_photo.JPG">
+              <br>
```

Comment:
The br tag is rarely what you want. Try to accomplish your goals with just CSS padding/margin. Actually, I removed this tag, and nothing changed, so I don't think it's doing what you think it's doing.

Reply:
will change this in another pull request since i remove this container in this branch